cypress/integration/ocmShared.js

```diff
@@ -0,0 +1,77 @@
+import { makeApiCall, API_BASE_URL } from './shared';
+import { DEFAULT_API_REQUEST_TIMEOUT } from './constants';
+
+export const INTEGRATION_ENV = Cypress.env('INTEGRATION_ENV');
+export const OCM_USER = Cypress.env('OCM_USER');
+export const OCM_USER_PASSWORD = Cypress.env('OCM_USER_PASSWORD');
+export const OCM_COOKIE_NAME = Cypress.env('OCM_COOKIE_NAME');
+export const OCM_TOKEN_DEST = Cypress.env('OCM_TOKEN_DEST');
+
+export const loginOCM = (userName, password) => {
+  // visit ocm environemnt based on INTEGRATION_ENV (true/false)
+  // cy.visit does not direct to the correct env based on baseUrl query string
+  const visitOptions = INTEGRATION_ENV ? { qs: { env: 'integration' } } : {};
+
+  cy.visit('', visitOptions);
+  cy.get('#username').should('be.visible');
+  cy.get('#username').type(userName);
+  cy.get('#username').should('have.value', userName);
+  cy.get('#login-show-step2').click();
+  cy.get('#password').should('be.visible');
+  cy.get('#password').type(password);
+  cy.get('#password').should('have.value', password);
+  cy.get('#kc-form-login').submit();
+
+  cy.get('#button-create-new-cluster', {
+    timeout: DEFAULT_API_REQUEST_TIMEOUT,
+  });
+};
+
+export const writeCookieToDisk = async (cookiename, cookiefile) => {
+  cy.getCookie(cookiename).then((cookie) => {
+    // dumping cookie to disk
+    cy.exec(`rm -f ${cookiefile}; echo ${cookie.value} >> ${cookiefile}`);
```

Comment:
nit: with replacing `>>` by single `>` you can skip the `rm`

Reply:
The `cookie.value` can theoretically contain. Wrapping in single quotes might help.